public/app/plugins/datasource/tempo/NativeSearch.tsx

```diff
@@ -53,6 +53,7 @@ const NativeSearch = ({ datasource, query, onChange, onBlur, onRunQuery }: Props
     spanNameOptions: [],
     selectedSpanName: undefined,
   });
+  const [validation, setValidation] = useState<{ [key: string]: boolean }>({});
```

Comment:
This for me is a little confusing, because we have value validation, but if I understood it correctly, it keeps value of `inavlidity`. Basically if `validation.minDuration` is `true`, then the value is invalid. Could we change it so when `validation.minDuration` is `true` the value is valid?

Reply:
I can see how that's confusing. I think 'errors' might make more sense as that's what a lot of form libraries (like react-hook-form which is used in some parts of the codebase) uses. I'll update that.

Reply:
Yeah, that is another good solution.